src/test/java/org/reaktivity/nukleus/maven/plugin/internal/generated/FlatFWTest.java

```diff
@@ -133,165 +230,172 @@ public void shouldFailToSetString1BeforeFixed1() throws Exception
+    {
+        buffer.putLong(offset, 10);
+        buffer.putShort(offset += 8, (short) 20);
+        buffer.putByte(offset += 2, (byte) "value1".length());
+        buffer.putBytes(offset += 1, "value1".getBytes(UTF_8));
+        buffer.putInt(offset+=6, 30);
+        buffer.putByte(offset += 4, (byte) "value2".length());
+        buffer.putBytes(offset += 1, "value2".getBytes(UTF_8));
+
+        return offset + 6;
     }
 
-    @Test
-    public void shouldSetStringValuesUsingBuffer() throws Exception
+    static FlatFW.Builder setAllFieldsValue(FlatFW.Builder builder)
     {
+        builder.fixed1(10);
+        builder.fixed2(20);
+        builder.string1("value1");
+        builder.fixed3(30);
+        builder.string2("value2");
+
+        return builder;
+    }
+
+    static FlatFW.Builder setAllFieldsValue1(FlatFW.Builder builder)
+    {
+        final StringFW.Builder stringRW = new StringFW.Builder();
+        final MutableDirectBuffer valueBuffer1 = new UnsafeBuffer(allocateDirect(100));
+        final MutableDirectBuffer valueBuffer2 = new UnsafeBuffer(allocateDirect(100));
+        StringFW value1 = stringRW.wrap(valueBuffer1,  0, valueBuffer1.capacity())
+                .set("value1", UTF_8)
+                .build();
+        StringFW value2 = stringRW.wrap(valueBuffer2,  0, valueBuffer2.capacity())
+                .set("value2", UTF_8)
+                .build();
+
+        builder.fixed1(10);
+        builder.fixed2(20);
+        builder.string1(value1);
+        builder.fixed3(30);
+        builder.string2(value2);
+
+        return builder;
+    }
+
+    static FlatFW.Builder setAllFieldsValue2(FlatFW.Builder builder)
+    {
+        final MutableDirectBuffer valueBuffer = new UnsafeBuffer(allocateDirect(100));
         valueBuffer.putStringWithoutLengthUtf8(0, "value1");
         valueBuffer.putStringWithoutLengthUtf8(10, "value2");
-        flatRW.wrap(buffer, 0, buffer.capacity())
-            .fixed1(10)
-            .fixed2(20)
-            .string1(valueBuffer, 0, 6)
-            .fixed3(30)
-            .string2(valueBuffer, 10, 6)
-            .build();
-        flatRO.wrap(buffer,  0,  100);
-        assertEquals(10, flatRO.fixed1());
-        assertEquals(20, flatRO.fixed2());
-        assertEquals("value1", flatRO.string1().asString());
-        assertEquals(30, flatRO.fixed3());
-        assertEquals("value2", flatRO.string2().asString());
+
+        builder.fixed1(10);
+        builder.fixed2(20);
+        builder.string1(valueBuffer, 0, 6);
+        builder.fixed3(30);
+        builder.string2(valueBuffer, 10, 6);
+
+        return builder;
     }
 
-    @Test
-    public void shouldSetStringValuesToNull() throws Exception
+    static FlatFW.Builder setRequiredFields(FlatFW.Builder builder, int toFieldIndex)
     {
-        flatRW.wrap(buffer, 0, buffer.capacity())
-            .fixed1(10)
-            .fixed2(20)
-            .string1((String) null);
+        if(toFieldIndex >= INDEX_FIXED1)
```

Comment:
remove the = in all conditions because toFieldIndex is EXCLUSIVE
also: space before (

Reply:
Done.